cmd/pbackend/statuser.go

```diff
@@ -374,7 +398,9 @@ func statuser() {
 	go checkSourceAvailabilityAzure(cancelCtx)
 
 	senderWG.Add(1)
-	go sendResults(cancelCtx, 1024, 5*time.Second)
+	// processing can be slowed down by configuration: send messages in maximum
+	// batch size of 8 messages or 4 seconds (what comes first)
+	go sendResults(cancelCtx, 8, 4*time.Second)
```

Comment:
Because we implemented slight slow-down (1 second) the buffer of 1k messages makes zero sense now, this is more tight configuration.